src/Deployer.php

```diff
@@ -77,7 +77,12 @@ public function addTarget(string $name, string $path, string $publicURL = null,
         $host = host($name)->setDeployPath($path);
 
         if (null !== $this->hostname) {
-            $host->setHostname($this->hostname);
+            [$hostname, $port] = explode(':', $this->hostname, 2);
```

Comment:
Not sure if we have strict type checking here, but this should prevent assignment error if `explode` only returns one value.

```suggestion
            [$hostname, $port] = explode(':', $this->hostname, 2) + [null, null];
```